proposal/matplotlib_cartopy.tex

```diff
@@ -423,128 +422,53 @@ \subsubsection{General maintenance}
 
 %plot?
 
-Historically, Pull Requests (PRs) and Issues have been submitted
+Historically, pull requests (PRs) and issues have been submitted
 faster than they can be reviewed; Matplotlib has accumulated about 300
-open PRs 1300 open Issues due to this imbalance, There almost
-certainly are critical bug reports or insightful feature requests among
+open PRs 1300 open issues due to this imbalance. There are critical bug reports and insightful feature requests among
 the issue backlog, while among the PR backlog there are useful contributions
 or bug fixes that would improve the libraries for direct users and
 downstream packages.  The large backlog is discouraging for new and
 occasional contributors and distracting for core developers.
 
-In 2020 Matplotlib resolved 125-200 PRs and 75-125 Issues a month,
-however even at this rate we only reduced the PR backlog by 70 open
-PRs and held even on the Issue backlog.  We were able to achieve this
-because for 10 months in 2020 we had supported RSE working on
-Matplotlib.  As part of reducing the backlog, we were able to fix a
-several long standing bugs.  We have demonstrated that supporting
+In 2020 Matplotlib had supported RSE [TODO?? What is an RSE???  Not defined] working on
+Matplotlib, during which time we resolved 125-200 PRs and 75-125 issues a month.
+However even at this rate we only reduced the PR backlog by 70 open
+PRs and only held even on the issue backlog.  As part of reducing the backlog, we were able to fix several long standing bugs.  We have demonstrated that supporting
 maintenance work has a positive effect on the health of the project
 and \textbf{the additional resources requested here will further
   reduce, but not eliminate, the maintenance backlog}.
 
-
 \subsection{Perceived impact of work}
 
+
 % TODO add cite for
 % \url{https://www.aosabook.org/en/matplotlib.html}
 
-Originally developed over 17 years ago the architecture of Matplotlib
```

Comment:
I moved some of the best of this up into 1.2.  I don't think we should rehash it.  Happy for other ideas for 1.3, but I think we pretty adequately argued the impact in the intro as well.